docs/reference/data-streams/logs.asciidoc

```diff
@@ -50,3 +50,196 @@ DELETE _index_template/my-index-template
 ----
 // TEST[continued]
 ////
+
+[[logsdb-default-settings]]
+
+[discrete]
+[[logsdb-synthtic-source]]
+=== Synthetic source
+
+By default, `logsdb` mode uses  <<synthetic-source,synthetic `_source`>>, which omits storing the original `_source`
+field and synthesizes it from doc values or stored fields upon document retrieval. Synthetic source comes with a few
+restrictions which you can read more about in the <<synthetic-source,documentation>> section dedicated to it.
+
+NOTE: When dealing with multi-value fields, the `index.mapping.synthetic_source_keep` setting controls how field values
```

Comment:
I am not sure if multi-value fields is clear enough. Maybe "when dealing with arrays of values"?

Reply:
In other places in the documents we use multi-value fields...which by the way is the correct name to distinguish. Elasticsearch doesn't  need to maintain array order because its core functionality revolves around searching based on the presence of values, not their position. Therefore, it treats arrays and multi-value fields as a set of independent values, where order doesn't play a role in indexing or querying. So, IMO it is where we use "array" that we make a mistake.

Reply:
Thanks for this context, sounds good.